res/text.f.glsl

```diff
@@ -24,15 +24,22 @@ uniform sampler2D mask;
 
 void main()
 {
-    if (backgroundPass != 0) {
+    if (pass == 0) {
+        // Background.
         if (bg.a == 0.0)
             discard;
 
         alphaMask = vec4(1.0);
         color = vec4(bg.rgb, 1.0);
-    } else {
+    } else if (pass == 1) {
+        // Regular text glyphs.
         vec3 textColor = texture(mask, TexCoords).rgb;
         alphaMask = vec4(textColor, textColor.r);
         color = vec4(fg, 1.0);
+    } else {
+        // Colored glyphs.
+        vec4 c = texture(mask, TexCoords);
```

Comment:
I'd be in favor of using name that makes a bit more sense than just `c`.

Reply:
Actually, we can remove this variable entirely (done in last version)